packages/blockchain/src/db/manager.ts

```diff
@@ -107,40 +104,6 @@ export class DBManager {
     if (hash === undefined || number === undefined) return undefined
     const header = await this.getHeader(hash, number)
     const body = await this.getBody(hash, number)
-    if (body[0].length === 0 && body[1].length === 0) {
```

Comment:
we don't need this reconstruction now because we now just simply write the body array as is